### DISCUSSION TITLE
History in Tab

### DISCUSSION THREAD

**Discussion by Michi-L** [sienori/Tab-Session-Manager#949] [Q&A]
Hi,

i just installed TSM and tried to save some sessions.
As I understood the description, it is saving the Sessions and not only the actually open Links.

Is my expectation right, that it should be possible to navigate backward and forward in the restored tabs, as it was before closing them?

Is it necessary to change the settings to get this function?

Best regards,
Michael

**Comment by rediffusion** [comment]
P.S.
It would be nice to have the possibility of adding tabs to the already existing session.

**Comment by AndreStarTrek** [comment]
Every tab session manager is missing this feature. Losing the history to move back and forward prevents me from using any manager. It must be possible because if you restore a tab session with chrome it self i get the history back.

If this function build in this extension, I would be happy to donate for this.

I hope to see this feature in the future ;)
